docs/pages/blog/mui-x-v5.md

```diff
@@ -0,0 +1,191 @@
+
+Some of the mentioned improvements were also applied to the rows, however the gains were more subtle.
+Besides the better performance, the new virtualization engine also brings the following fixes:
+
+- Horizontal and vertical scroll share the same logic.
+- No more jumps when changing the rendered rows.
+- Calling `apiRef.current.scrollToIndexes` works no matter where the cell is.
+- Improved support for when `disableVirtualization` is used.
+- Fixes keyboard navigation with arrow keys.
+
+## Improved state management
+
+Several enhancements were made to state management to improve developer experience, performance and consistency in the execution order.
+
+#### Improving the developer experience around the state
+
+To improve developer experience for both the X-team and developers using the `apiRef` features, we are working on making the state structure and the tools to access it as easy to understand as possible.
+
+- We removed the `state` structure from the public API. Access to data in the state should always be done through `apiRef` methods (`apiRef.current.getSelectedRows`) or selectors (`selectedGridRowsSelector`).
+- We renamed most selectors to have a consistent naming convention, making it easier to deduce their name or infer purpose.
+- We restructured our state so that each feature has a single substate, and is the only one to update it (`state.filter` is only managed by our `useGridFilter` hooks which exposes methods for both internal and 3rd party code to interact with this state).
+
+The work on that topic is far from over. We have several developments in progress or under discussion to improve the developer experience when using the advanced features of the grid.
+Here are a few that should be release in the following months:
+
+- Strict typing of event listeners and publishers.
+- Examples for the event listeners.
+- Documentation and examples for selectors.
+- Add the ability to export and restore some parts of the grid state.
+
+#### Synchronous state initialization
+
+In previous versions, the state was first populated with default values, and then in a `useEffect`, given the values provided as props (`props.pageSize` for instance), or derived from the props (the sorted and filtered rows derived from the `props.rows`, `props.sortModel` and `props.filterModel`).
+This was causing an additional render with useless data, and the we had to be careful to avoid flickering between the fake and real data.
+In v5, the state is initialized synchronously during the first render.
+
+Note that for now, the state updates coming from controlled props are still asynchronous.
+If you pass `props.pageSize`, we will apply it to the state in a `useEffect`, and therefore if you read the state just after the render (for instance in a `useLayoutEffect`), you will still see the old version.
+
+## Reduced style specificity for easier customization
```

Comment:
Could this header be too long? Instead of easier, maybe simpler would be even better (simpler/complex !== easier/harder). 

```suggestion
## Simpler style customization
```